plugins/postgres/dbt/adapters/postgres/relation.py

```diff
@@ -1,4 +1,29 @@
 from dbt.adapters.base import Column
+from dataclasses import dataclass
+from dbt.adapters.base.relation import BaseRelation
+from dbt.exceptions import RuntimeException
+
+
+@dataclass(frozen=True, eq=False, repr=False)
+class PostgresRelation(BaseRelation):
+    def __post_init__(self):
+        # Check for length of Postgres table/view names.
+        # Check self.type to exclude test relation identifiers
+        if (
+            self.identifier is not None
+            and self.type is not None
+            and len(self.identifier) > self.relation_max_name_length()
+        ):
+            raise RuntimeException(
+                "Postgres relation name '"
+                + self.identifier
+                + "' is longer than "
+                + str(self.relation_max_name_length())
+                + " characters"
+            )
```

Comment:
Instead of string additions, use format strings. `f-strings` are powerful and readable:
```python
f"Postgres relation name '{self.identifier}' is longer than {self.relation_max_name_length()} characters"
```

That's probably too long, but you can also use python's behavior of automatically concatenating strings instead of having to add them:
```python
                f"Postgres relation name '{self.identifier}' is longer than "
                f"{self.relation_max_name_length()} characters"
```

Reply:
Much nicer. Thanks!